README.md

```diff
@@ -455,11 +456,29 @@ Dodge et. al 2015. Tests Memory Networks on 4 tasks including reddit dialog task
 
 * [Hindi Dependency Treebank](http://ltrc.iiit.ac.in/treebank_H2014/) - A multi-representational multi-layered treebank for Hindi and Urdu
 * [Universal Dependencies Treebank in Hindi](http://universaldependencies.org/treebanks/hi/index.html)
-     * [Parallel Universal Dependencies Treebank in Hindi](http://universaldependencies.org/treebanks/hi_pud/index.html) - A smaller part of the above-mentioned treebank.
+     * [Parallel Universal Dependencies Treebank in Hindi](http://universaldependencies.org/treebanks/hi_pud/index.html) - A smaller 
+     part of the above-mentioned treebank.
+     
+## NLP in Thai
+
+[Back to Top](#contents)
+
+### Libraries
+
+* [PyThaiNLP](https://github.com/wannaphongcom/pythainlp) - Thai NLP in Python Package
+* [JTCC](https://github.com/wittawatj/jtcc)- A character cluster library in Java
+* [SWATH](http://www.cs.cmu.edu/~paisarn/software.html) - Word segmentation in C language
+* [CutKum](https://github.com/pucktada/cutkum) - Word segmentation with deep learning in TensorFlow
+* [Thai Language Toolkit](https://pypi.python.org/pypi/tltk/) - Based on a paper by Wirote Aroonmanakun in 2002 with included dataset
+* [SynThai](https://github.com/KenjiroAI/SynThai)- Word segmentation and POS tagging using deep learning in Python
+
+### Corpora
+
+* [Thai Wikipedia](https://dumps.wikimedia.org/thwiki/latest/thwiki-latest-pages-articles.xml.bz2)- A dataset of 1.49GB in XML format
```

Comment:
We don't list Wiki corpora because that's standard corpora and _almost everyone_ knows about. 

For Thai in particular, please replace with NECTEC's BEST dataset link as the better reference point.